README.md

```diff
@@ -1,2 +1,25 @@
-# Okteto desk
-Your desktop extended into your kubernetes cluster
\ No newline at end of file
+# Okteto Desk: your desktop extended into your kubernetes cluster
```

Comment:
I think "desktop" is misleading because it represents a very established metaphor for something that has nothing to do with this. I like the term "desk" because it represents your working area (no need for the "-top").

Try: `Okteto Desk: your work environment extended into your Kubernetes cluster`